lib/systems/examples.nix

```diff
@@ -82,6 +82,12 @@ rec {
     libc = "bionic";
     useAndroidPrebuilt = false;
     useLLVM = true;
+    cc = "clang";
+    bintools = "llvm";
+    cxxlib = "libcxx";
+    cxxrtlib = "libcxxabi";
+    unwindlib = "libunwind";
+    rtlib = "compiler-rt";
```

Comment:
I don’t think we should set these while they’re redundant to `useLLVM` (since they couldn’t be any other possible value, and adjusting them won’t work, and it means that e.g. `lib.systems.examples.aarch64-freebsd // { useLLVM = false; }` becomes broken for no good reason).